openslides/core/models.py

```
@@ -252,3 +253,19 @@ class Meta:
 
     def __str__(self):
         return 'Message {}'.format(self.timestamp)
+
+
+class Session(DjangoSession):
+    """
+    Model like the Django db session, which saves the user as ForeignKey instead
+    of an encoded value.
+    """
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+        null=True)
+
+    @classmethod
+    def get_session_store_class(cls):
+        from .session import SessionStore
```

This should raise an Import Error.